src/app/users/views/summaryPartial.ejs

```diff
@@ -69,17 +84,22 @@
                 <li><a href="web-service-sync">Send WS Sync</a></li>
             </ul>
         </aside>
-        <% } else if (locals.isInvitation && !locals.user.deactivated) { %>
+        <% } else if (locals.isInvitation) { %>
         <aside>
             <h2 class="heading-medium">Actions</h2>
             <ul class="list">
-                <li><a href="associate-organisation">Add organisation</a></li>
-                <% if (locals.organisations.length > 0) { %>
-                    <li><a href="select-organisation">Add services</a></li>
+                <% if (!locals.user.deactivated) { %>
+                    <li><a href="associate-organisation">Add organisation</a></li>
+                    <% if (locals.organisations.length > 0) { %>
+                        <li><a href="select-organisation">Add services</a></li>
+                    <% } %>
+                    <li><a href="edit-email">Change user's email address</a></li>
+                    <li><a href="confirm-invitation-deactivation">Deactivate user</a></li>
+                    <li><a href="resend-invitation"> Resend invitation email</a></li>
+                <% } %>
+                <% if (locals.user.deactivated) { %>
```

Comment:
This is only relevant for users so is not needed in this block, you can reactive in the banner which is fine

Reply:
Removed and reverted back to original sidebar